convert2rhel/actions/system_checks/duplicate_packages.py

```diff
@@ -31,7 +31,7 @@ def run(self):
         super(DuplicatePackages, self).run()
 
         logger.task("Prepare: Check if there are any duplicate installed packages on the system")
-        output, _ = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
+        output, ret = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
```

Comment:
This is unrelated right? If `ret` isn't used I'd vote for `_`

Reply:
So, I was discussing this with Preston last week and I asked him to check for return code in this command instead of just strings.. I think the `ret` is for him to test if there's any return code in case of failure that we can use to identify this specific problem

Reply:
Yes that was leftover from testing, i'll revert it to _